src/panels/config/automation/blueprint-automation-editor.ts

```diff
@@ -0,0 +1,293 @@
+        ${!this.narrow
+          ? html` <span slot="header">${this.config.alias}</span> `
+          : ""}
+        <span slot="introduction">
+          ${this.hass.localize(
+            "ui.panel.config.automation.editor.introduction"
+          )}
+        </span>
+        <ha-card>
+          <div class="card-content">
+            <paper-input
+              .label=${this.hass.localize(
+                "ui.panel.config.automation.editor.alias"
+              )}
+              name="alias"
+              .value=${this.config.alias}
+              @value-changed=${this._valueChanged}
+            >
+            </paper-input>
+            <paper-textarea
+              .label=${this.hass.localize(
+                "ui.panel.config.automation.editor.description.label"
+              )}
+              .placeholder=${this.hass.localize(
+                "ui.panel.config.automation.editor.description.placeholder"
+              )}
+              name="description"
+              .value=${this.config.description}
+              @value-changed=${this._valueChanged}
+            ></paper-textarea>
+          </div>
+          ${this.stateObj
+            ? html`
+                <div class="card-actions layout horizontal justified center">
+                  <div class="layout horizontal center">
+                    <ha-entity-toggle
+                      .hass=${this.hass}
+                      .stateObj=${this.stateObj!}
+                    ></ha-entity-toggle>
+                    ${this.hass.localize(
+                      "ui.panel.config.automation.editor.enable_disable"
+                    )}
+                  </div>
+                  <mwc-button
+                    @click=${this._excuteAutomation}
+                    .stateObj=${this.stateObj}
+                  >
+                    ${this.hass.localize("ui.card.automation.trigger")}
+                  </mwc-button>
+                </div>
+              `
+            : ""}
+        </ha-card>
+      </ha-config-section>
+
+      <ha-config-section .isWide=${this.isWide}>
+        <span slot="header">Blueprint</span>
+        <ha-card>
+          <div class="card-content">
+            <div class="blueprint-picker-container">
+              ${this._blueprints
+                ? html`
+                    <ha-blueprint-picker
+                      .hass=${this.hass}
+                      label="Blueprint to use"
+                      .blueprints=${this._blueprints}
+                      .value=${this.config.use_blueprint.path}
+                      @value-changed=${this._blueprintChanged}
+                    ></ha-blueprint-picker>
+                  `
+                : "You don't have any blueprints."}
+              <mwc-button @click=${this._navigateBlueprints}>
+                Manage Blueprints
+              </mwc-button>
+            </div>
+            ${this.config.use_blueprint.path
+              ? html` ${blueprint?.metadata?.input &&
```

Comment:
Does this html tag only have a ternary statement ?

Reply:
Also, I personally prefer if we would try to have a newline after the html tag. Indentation makes it slightly easier.